mmv1/third_party/terraform/website/docs/r/logging_project_sink.html.markdown

```diff
@@ -18,6 +18,9 @@ Manages a project-level logging sink. For more information see:
 
 ~> **Note** You must [enable the Cloud Resource Manager API](https://console.cloud.google.com/apis/library/cloudresourcemanager.googleapis.com)
 
+~> **Note:** Logging sinks are automatically created for a given project and cannot be deleted. Creating a resource of this type will acquire and update the resource that already exists at the desired location. These sinks cannot be removed so deleting this resource will remove the sink config from your terraform state but will leave the logging sink unchanged. The sinks that are currently automatically created are "_Default" and "_Required".
```

Comment:
I found this slightly unclear, because at the beginning of this note it sounds like _all_ logging sinks are automatically created and _none_ of them can be removed. Following the pattern used by google_logging_project_bucket_config, an alternative might be to add something like `Logging automatically creates two sinks: _Required and _Default.` at the end of the `name` description.